### ISSUE TITLE
std::string_view not recognized

### ISSUE THREAD

**Issue by Arthapz** [standardese/standardese#183]
Hi,
i have in a standardese.ini with
```
[compilation]
standard=c++17
```

but standardese give me errors with std::string_view

```
[libclang] [error] /home/arthapz/builds/stormkit/debug/core/Configure.hpp:13: no member named 'string_view' in namespace 'std'
[libclang] [error] /home/arthapz/builds/stormkit/debug/core/Configure.hpp:17: no member named 'string_view' in namespace 'std'
[libclang] [error] /home/arthapz/builds/stormkit/debug/core/Configure.hpp:18: no member named 'string_view' in namespace 'std'
[libclang] [error] /home/arthapz/builds/stormkit/debug/core/Configure.hpp:19: no member named 'string_view' in namespace 'std'
[libclang] [error] /home/arthapz/builds/stormkit/debug/core/Configure.hpp:20: no member named 'string_view' in namespace 'std'
```

the file is

```cpp
#pragma once

#include <string_view>

static constexpr auto STORM_VERSION = std::string_view{"0.2.0"};
static constexpr auto STORM_MAJOR_VERSION = 0;
static constexpr auto STORM_MINOR_VERSION = 2;
static constexpr auto STORM_PATCH_VERSION = 0;
static constexpr auto STORM_BUILD_TYPE = std::string_view{"debug"};
static constexpr auto STORM_GIT_COMMIT_HASH = std::string_view{"fc235d5"};
static constexpr auto STORM_GIT_BRANCH = std::string_view{"4-write-documentation"};
``` 

from what i see, standardese is in c++14 mode (other errors or warning like ''auto not allowed in template parameter until C++17")

i tried with c++1z value, but got a segfault

**Comment by foonathan** [comment]
This should be fixed once foonathan/cppast#115 is merged and standardese updated.

**Comment by foonathan** [comment]
The PR is now merged and the file can be parsed on cppast's end.

**Comment by Curve** [comment]
I'm not sure if it's a misconfiguration on my side but I'm also having problems with std::optional

```
[libclang] [error] webviewpp/build/../webview/include/javascript/promise.hpp:20: no template named 'optional' in namespace 'std'
```

**Comment by foonathan** [comment]
This should be fixed as soon as stabdardese updates cppast.

**Comment by saraedum** [comment]
If someone open a PR with the necessary upgrades for this, I am happy to merge it and create a release

**Comment by saraedum** [comment]
Fixed in #200
